build.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
-
-
 set -e
 echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-docker build -t $DOCKER_ORG/init-controller:$TRAVIS_TAG -t $DOCKER_ORG/init-controller:latest .
-docker push $DOCKER_ORG/init-controller
+export DOCKER_CLI_EXPERIMENTAL=enabled
```

Comment:
It would be create to add some comments to explain this process. What is the minimum Docker version that supports `buildx`?